src/main/scala/io/iohk/praos/domain/StakeholderState.scala

```diff
@@ -0,0 +1,18 @@
+package io.iohk.praos.domain
+
+import io.iohk.praos.crypto.{Hasher, Key}
+
+
+object StakeholderState {
+  // This function corresponds to the initStakeholderStake function in the Praos Formalization.
+  def initialize(privateKey: Key, genesisBlock: GenesisBlock): StakeholderState =
+    StakeholderState(privateKey, genesisBlock, genesisBlock, Blockchain(), None, List.empty)
+}
+
+case class StakeholderState(
```

Comment:
Why is this needed? Maybe I'm missing something but it doesn't seem to be currently used. Also check `BlockchainState` which already has part of the information from this class.

Reply:
Correct, removed.